content/build/development-guides/adding-an-environment-to-the-portability-service.md

```diff
@@ -0,0 +1,25 @@
+---
+path: "/build/development-guides/adding-an-environment-to-the-portability-service.md"
+date: "2018-07-12"
+title: "Adding an Environment to the Portability Service"
+---
+
+## Why add an environment to the portability service?
+
+HCA pipelines and other user-submitted pipelines are executed in multiple environments during development and release to demonstrate their portability. The environments in which these pipelines run benefit in two ways:
```

Comment:
This page seems like a good addition to me.  Just a couple of small comments:
change this sentence to say: This testing is beneficial in two respects